cr3qt/data/docx.css

```diff
@@ -14,45 +14,45 @@ li image { display: inline }
 li { display: list-item; text-indent: 0em;  }
 ol { display: block; list-style-type: decimal; margin-left: 1em }
 
-title.h1 p, title.h2 p { 
+.h1, .h2 {
     $title.all
 }
 
-title.h3 p, title.h4 p, title.h5 p, title.h6 p { 
+.h3, .h4, .h5, .h6 p {
```

Comment:
Probably typo in these (you removed the "p" in the "hx p" descendant rules. Either that's what you wanted and you forgot to remove it in the `h6 p`, or it's not and you removed too many (with `h3, h4, h5, h6 p`, you get 4 selectors: `h3` `h4` `h5` `h6 p` - it does _not_ become `h3 p` `h4 p` `h5 p` `h6 p`)

Reply:
Thanks @poire-z, indeed I forgot remove it in the h6 p